Bottom: 6794b4a32934bdb01982d1a31f6d40e6b7ba2634
Top:    a49b8d132b5f15902dc7ad1518793970f32e3fd8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-02 17:18:01 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 3f3c5d0..4ab6e37 100644
--- a/configure.ac
+++ b/configure.ac
@@ -360,8 +360,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
